src/routes/tsoa/login.ts

```diff
@@ -31,6 +31,7 @@ import { applyTokenResponse } from "../../helpers/apply-token-response";
 import { sendResetPassword } from "../../controllers/email";
 import { validateCode } from "../../authentication-flows/passwordless";
 import { UniversalLoginSession } from "../../adapters/interfaces/UniversalLoginSession";
+import { getUsersByEmail } from "../../utils/users";
```

Comment:
Added a helper for this so we eventually can remove the getUsersByEmail from the adapters